packages/react/src/wallet/useProfilesManaged.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  useProfilesManaged as useProfilesManagedHook,
+  ProfilesManagedRequest,
+  Profile,
+} from '@lens-protocol/api-bindings';
+
+import { useLensApolloClient } from '../helpers/arguments';
+import { PaginatedArgs, PaginatedReadResult, usePaginatedReadResult } from '../helpers/reads';
+
+/**
+ * {@link useProfilesManaged} hook arguments
+ */
+export type UseProfilesManagedArgs = PaginatedArgs<ProfilesManagedRequest>;
```

Comment:
`ProfilesManagedRequest` includes too many low level features including `cursor` with the odd codegen typings `?: InputMaybe<...>`.

Can we list what relevant?

Reply:
hm, `PaginatedArgs<>` removes the cursor and we use the same approach in every other paginated hook, what is different here?

Reply:
I forgot 😅 ... all good!